packages/web-components/src/components/va-date/va-date.tsx

```diff
@@ -205,23 +220,25 @@ export class VaDate {
                   <option value={month.value}>{month.label}</option>
                 ))}
             </va-select>
-            <va-select
-              label="Day"
-              name={`${name}Day`}
-              // If day value set is greater than amount of days in the month
-              // set to empty string instead
-              // Value must be a string
-              value={daysForSelectedMonth.length < day ? '' : day?.toString()}
-              onVaSelect={handleDateChange}
-              class="select-day"
-              aria-label="Please enter two digits for the day"
-            >
-              <option value=""></option>
-              {daysForSelectedMonth &&
-                daysForSelectedMonth.map(day => (
-                  <option value={day}>{day}</option>
-                ))}
-            </va-select>
+            {!monthYearOnly && (
+              <va-select
```

Comment:
The only thing changing here besides the `!monthYearOnly` conditional is the indentation.